stdlib/float.mli

```diff
@@ -225,6 +276,14 @@ external copysign : float -> float -> float
     If [y] is [nan], returns either [x] or [-. x], but it is not
     specified which. *)
 
+external signbit : (float [@unboxed]) -> bool
```

Comment:
Same as above here. Note that Gg.Float uses [`sign_bit`](https://github.com/dbuenzli/gg/blob/4d4c214f9c8ad8790a6dcca7f6fcb3c263dd16e1/src/gg.mli#L178-L179).